cmd/flags.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"errors"
+	"strings"
+)
+
+// ErrorEmpty returned when value is empty.
+var ErrorEmpty = errors.New("empty value not allowed")
+
+// StringSlice is a flag that allows to set multiple types of string
```

Comment:
I don't really get this docstring. `StringSlice is a type of flag that allows setting multiple string values.`?

Reply:
yes. sorry i will reword